File: python/021.py
# 21) Faça um algoritmo que leia um determinado ano e mostre se ele é ou não
# BISSEXTO.

pegar_ano = int(input("Digite o ano para ver se ele é BISSEXTO ou NÃO: "))
if pegar_ano % 4 == 0:
    print(f"O ano {pegar_ano} é BISSEXTO!")
else:
    print(f"O ano {pegar_ano} não é BISSEXTO!")

File: javascript/021.js
/*
21) Faça um algoritmo que leia um determinado ano e mostre se ele é ou não
BISSEXTO.
*/

const isBissexto = (ano) => {
  if (ano % 4 === 0) return "É BISSEXTO";
  else return "Não é BISSEXTO";
};

console.log(isBissexto(2002));
console.log(isBissexto(1988));
console.log(isBissexto(1996));
console.log(isBissexto(2005));
console.log(isBissexto(2000));
